File: python_vs_golang/age_category.go
package main

import "fmt"
func Age_Category(age int) string {
	if age < 0 {
		return "Invalid age"
	} else if age < 13 {
		return "Child"
	} else if age < 18 {
		return "Teenager"
	} else if age < 65 {
		return "Adult"
	}
	return "Senior"
}
func main() {
	fmt.Println(Age_Category(17))
}

File: python_vs_golang/age_category.py
def Age_category(age):
    if age < 0:
        return "Invalid age"
    elif age < 13:
        return "Child"
    elif age < 18:
        return "Teenager"
    elif age < 65:
        return "Adult"
    else:
        return "Senior"

print(Age_category(18))
print(Age_category(70))
print(Age_category(14))